src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private final double longerSide;
+    private final double height;
+
+    public Rectangle(double longerSide, double height, String color) {
+        super(color);
+        this.longerSide = longerSide;
+        this.height = height;
+    }
+
+    @Override
+    public double getArea() {
+        return longerSide * height;
+    }
+
+    @Override
+    public void toDraw() {
+        System.out.println("Figure: rectangle, area: " + getArea() + " sq"
+                + ", longer side " + longerSide + " units "
+                + ", height: " + height + " units "
+                + " color: " + color);
```

Comment:
Instead of directly printing from the `toDraw()` method, consider following the Single Responsibility Principle. This method could return the drawing details as a String, and the actual printing could be handled elsewhere.